pkg/handler3/handler.go

```diff
@@ -217,6 +219,14 @@ func (o *OpenAPIService) HandleGroupVersion(w http.ResponseWriter, r *http.Reque
 			if err != nil {
 				return
 			}
+			// Set Content-Type header in the reponse
+			if accepts.SubType == subTypeProtobufDeprecated {
```

Comment:
same comment about putting `ReturnedContentType string` in the `accepted` items above and making this an unconditional `w.Header().Set("Content-Type", accepts.ReturnedContentType)`